tests/end2end/test_public_reset_password.py

```diff
@@ -141,3 +141,42 @@ def test_sees_error_message_if_attempting_to_use_reset_code_twice(password_reset
     # Sees error.
     assert escape(_('Reset code "%(code)s" already used (%(isoformat)s)', code=password_reset.code,
                     isoformat=password_reset.modified_at.isoformat() + 'Z')) in res
+
+
+def test_sees_error_message_if_too_many_active_password_resets(db, testapp):
+    """Show error if too many active password resets exist."""
+    inactive_user = UserFactory(is_active=False)
```

Comment:
I think it's a bit confusing explicitly making the user inactive for this test case. Makes one wonder, why so?

Easier to understand the test case if it contains the bare minimum for satisfying a requirement; "error message instead of email if too many active (and also recent) password resets are pending".

(In the test case this was copy-pasted from, the purpose of it being inactive (initially) was to assert that after completing the request reset -> click link -> fill in new password workflow, the account would change state to active.)

Proposing to ...
1. add any user via the arg fixture, `..._active_password_resets(user, testapp)`
2. remove stuff related to user being active/inactive

Reply:
Copy-paste from `test_can_complete_password_reset_flow(db, testapp)`. I'll clean it up!